test/parallel/test-console-methods.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+require('../common');
+
+// This test ensures that console methods
+// cannot be invoked as constructors
+
+const assert = require('assert');
+
+const { Console } = console;
+const newInstance = new Console(process.stdout);
+const errMsg = /is not a constructor/;
+
```

Comment:
☝️ Testing for a engine specific error message. Instead it could just check for the TypeError.

Reply:
Fixed.